### ISSUE TITLE
Warn about scatter(colors=...)

### ISSUE THREAD

**Issue by davidwagner** [data-8/datascience#386]
Once we've released the version with supports for scatter(group=) and have converted assignments and textbook (PR #384), let's warn on use of scatter(color=).  See https://github.com/data-8/datascience/pull/384#discussion_r296495737.

**Comment by adnanhemani** [comment]
I've seen some commits to the Textbook regarding the API change. Have we done this to the assignments yet?

**Comment by davidwagner** [comment]
Textbook change is done.  Haven't made the changes to assignments yet.

**Comment by davidwagner** [comment]
Both changes are now done.

**Comment by davidwagner** [comment]
I propose that the next step is to create a pull request that deprecates `scatter(colors=)` and warns on use of it, but still supports that functionality for now (so we don't break older code).

**Comment by adityakuppa26** [comment]
Fixed the issue in https://github.com/data-8/datascience/pull/451

**Comment by adnanhemani** [comment]
Closed as per #389 .

**Comment by mycarta** [comment]
Hi there
FYI `colors` is still used in the notebooks for the edX  Data8.1x course (for example in the cde cell below, from lab 05 :

```
from functools import lru_cache as cache
# This cache annotation makes sure that if the same year
# is passed as an argument twice, the work of computing
# the result is only carried out once and then saved.
@cache(None)
def stats_relabeled(year):
    """Relabeled and cached version of stats_for_year."""
    return stats_for_year(year).relabeled(2, 'Children per woman').relabeled(3, 'Child deaths per 1000 born')
def fertility_vs_child_mortality(year):
    """Draw a color scatter diagram comparing child mortality and fertility."""
    with_region = stats_relabeled(year).join('geo', countries.select('country', 'world_6region'), 'country')
    with_region.scatter(2, 3, sizes=1, colors=4, s=500)
    plots.xlim(0,10)
    plots.ylim(-50, 500)
    plots.title(year)
fertility_vs_child_mortality(1960)
```
I just completed that class and downloaded the lab notebooks for further play unning them locally on my machine. For that purpose I created a virtual env with the latest version of the `datascience` library (updated today to be sure). I presume the notebooks running online use an older version because I get an error, but was not getting it when worked on the lab online.

I fixed it by replacing this line:
`with_region.scatter(2, 3, sizes=1, colors=4, s=500)`
with this one:
`with_region.scatter(2, 3, sizes=1, group=4, s=500)`

I posted this also on the lab discussion board.
